pkg/statistics/table.go

```diff
@@ -122,53 +128,50 @@ func (m *ColAndIdxExistenceMap) HasAnalyzed(id int64, isIndex bool) bool {
 }
 
 // InsertCol inserts a column with its meta into the map.
-func (m *ColAndIdxExistenceMap) InsertCol(id int64, info *model.ColumnInfo, analyzed bool) {
-	m.colInfoMap[id] = info
+func (m *ColAndIdxExistenceMap) InsertCol(id int64, analyzed bool) {
```

Comment:
Add a `intest.Assert(m.checked)`?

Reply:
It might be either false or true. When you initialize, it must be false, but if you just update, then it might be true.